.github/workflows/ci_build_wheels.yml

```
@@ -39,7 +39,7 @@ jobs:
           - os: ubuntu-20.04
             system_type: ["manylinux", "2014"]
             arch: x86_64
-            py_minors: 7,10
+            py_minors: 7,8,9,10,11
```

```suggestion
            py_minors: 7,11
```
This is just the test workflow.
Look into the [container_builds_weekly.yml](https://github.com/SINTEF/dlite/blob/master/.github/workflows/container_builds_weekly.yml) file, as well as possibly  [pyproject.toml](https://github.com/SINTEF/dlite/blob/master/python/pyproject.toml) or [setup.py](https://github.com/SINTEF/dlite/blob/master/python/setup.py) if any changes should be necessary.